src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js

```diff
@@ -78,6 +81,29 @@ const ValidatorsMonitor = ({ watchList }) => {
     config: { params: { address } },
   });
   const isValidator = validators?.data?.length > 0 && !isLoadingValidators;
+  const {
+    appEvents: {
+      transactions: { rewards },
+    },
+  } = useContext(ApplicationBootstrapContext);
+  const notification = rewards.length && BigInt(rewards[0]?.reward || 0) > BigInt(0);
+
+  useEffect(() => {
```

Comment:
If we put this effect in this component, would this notification still be shown if the user only stays on the /wallet page? 🤔 

Generally it feels like this useEffectHook should be placed in `ApplicationBootstrap` (In in another top application layer). Similar to how we add `useLedgerDeviceListener()` in  `ApplicationBootstrap`... For example a hook,
`useUnclamedRewardsToaster` or something.

Reply:
The notification symbol would still be shown on the side bar and if the user clicks on the Validators in the side bar they would then get the toast which would need to be manually closed even if the user navigates to another page. Having it done further up the layer would have been perfect if it was the only form of notification